### ISSUE TITLE
Wiki: Project Profile Issue Template

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#5630]
### Overview
We need a wiki page that explains how to make a developer issue out of a Project Profile Change Request Issue created by Stakeholder so that when these requests come in, it's easy for anyone on the merge team to turn it into an actionable issue, without prior knowledge of how those issues get built.

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

### Jessica's notes
- Please leave the "Complexity: Missing" and "size: missing" labels for now.
- This is currently still a work in progress.
- The purpose of this is
  - to have a template that can be used to easily write issues from "Update Project Profile" requests
  - to test out a possible format for the Project Profile Template in the wiki, where each comment below will represent a page in the wiki
- The content in this may eventually get moved elsewhere.

**Comment by JessicaLucindaCheng** [comment]
# Project Profile Issue Template
This wiki contains templates and examples for helping to write an issue to update a project's profile.

## Instructions
Please read our [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki page, if you haven't already.

Copy and paste the templates from below. Notice any instructions in brackets `[   ]` and replace the instructions (including the brackets) as instructed. For example, by following the instructions in `Update Project Profile: [Insert project's name] ([Insert short description of change])`, the result can be `Update Project Profile: Lucky Parking (Add Danny Pham)`.

## <a name="table-of-contents">Table of Contents</a>
1. [Add a title to the issue](#title)
2. [Add the basic issue format](#section-2)(https://github.com/hackforla/website/issues/5630#issuecomment-1740263532)
3. Based on the requested changes, add the relevant action items from below
  3.1. [Update project title](#section-3)(https://github.com/hackforla/website/issues/5630#issuecomment-1774117138)
  3.2. [Update the description](https://github.com/hackforla/website/issues/5630#issuecomment-1774119160)
  3.3. Update the image
---> Add Update image hero here
  3.4. [Update the technologies](https://github.com/hackforla/website/issues/5630#issuecomment-1774119289)
  3.5. [Update the partner(s)](https://github.com/hackforla/website/issues/5630#issuecomment-1774119349)
  3.6. [Update the tools](https://github.com/hackforla/website/issues/5630#issuecomment-1774119429)
  3.7. [Update the GitHub link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119521)
  3.8. [Update the Slack link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119587)
  3.9. [Update the Test Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119650)
  3.10. [Update Demo Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119730)
  3.11. [Update Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119764)
  3.12. [Leadership](https://github.com/hackforla/website/issues/5630#issuecomment-1774122890)
      3.12.1. Remove person(s) and info from leadership
      3.12.2. Add person(s) and info to leadership
  3.13. [Change project status](https://github.com/hackforla/website/issues/5630#issuecomment-1797822821)
4. Add labels
5. Add milestone
6. Add parent
7. [Submit new issue](https://github.com/hackforla/website/issues/5630#issuecomment-2003436167)
8. [Check the issue is added to the "Project Board (hackforla/website)"](https://github.com/hackforla/website/issues/5630#issuecomment-2003566185)

**Comment by JessicaLucindaCheng** [comment]
## 1. Add a title to the issue

### Template
```
Update Project Profile: [Insert project's name] ([Insert short description of change])
```

### Example
- https://github.com/hackforla/website/issues/5792 

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
## 2. Add the basic issue format

### Template
```
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] [Insert relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
  - [ ] the project's card on the Projects' page [^3]   [If the ONLY changes are for the `leadership` variable, REMOVE this action item to check the project's card.]

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project-specific page URL here]
[^3]: Projects page URL: https://www.hackforla.org/projects/ [If the ONLY changes are for the `leadership` variable, REMOVE this resource to the Projects page.]
```

### Example for Leadership Only Changes
- https://github.com/hackforla/website/issues/6632 

### Example for All Other Changes
- https://github.com/hackforla/website/issues/5790 

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.1. Update project title

## Template
````
- [  ] Find the `title` variable and change the following:

**From:**
```
[Insert current line(s) of code for title]
```
**To:**
```
[Insert desired code changes to title]
```
````

## Example
- **[PENDING]** When there is a title change for a project, add that issue as an example here

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.2. Update the description

## Template
````
- [  ] Find the `description` variable and change the following:

**From:**
```
[Insert current line(s) of code for description]
```
**To:**
```
[Insert desired code changes to description]
```
````

## Example
- **[PENDING]** When there is a description change for a project, add that issue as an example here

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.3. Update the image (WIP)

## Template
````
- [  ] Find the `description` variable and change the following:

**From:**
```
[Insert current line(s) of code for description]
```
**To:**
```
[Insert desired code changes to description]
```
````

## Example
- **[PENDING]** When there is an image change for a project, add a link to that issue as an example here.


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.4. Update the technologies
## 3.4.1. Remove a technology template
````
- [ ] Find the `technologies` variable and remove the following:
```
[Insert the line of code to remove the specified technology]
```
````
## 3.4.2 Add a technology template
````
- [ ] Find the `technologies` variable and add the following:
```
[Insert the line of code to add the desired technology]
```
````
## Example
- #5854 


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.5. Update the partner(s)
## Template
````
- [ ] Find the `partner` variable and change the following:

**From:**
```
[Insert current line of code for partner]
```
**To:**
```
[Insert desired code changes to partner]
```
````

## Example
- [ ] Find the `partner` variable and change the following:

**From:**
```
partner: Department of Neighborhood Empowerment
```
**To:**
```
partner: Department of Neighborhood Empowerment, Atwater Village
```


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.6. Update the tools

## Template
````
- [ ] Find the `tools` variable and change the following:

**From:**
```
[Insert current lines of code for tools]
```
**To:**
```
[Insert desired code changes to tools]
```
````

## Example
- [ ] Find the `tools` variable and change the following:

**From:**
```
tools:
  - Google Colab
```
**To:**
```
tools:
  - pnpm
  - GitHub
  - Storybook
  - Google Colab
```


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.7. Update the GitHub link

## Template
````
- [  ] Under the `links` variable, find `name: GitHub` and change the following:

**From:**
```
  - name: GitHub
    url: `[Insert current URL]`
```
**To:**
```
  - name: GitHub
    url: `[Insert desired URL]`
```
````

## Example
- **[PENDING]** When there is a GitHub link change for a project, add that issue as an example here


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.8. Update the Slack link

## Template
````
- [  ] Under the `links` variable, find `name: Slack` and change the following:

**From:**
```
  - name: Slack
    url: `[Insert current URL]`
```
**To:**
```
  - name: Slack
    url: `[Insert desired URL]`
```
````

## Example
- **[PENDING]** When there is a Slack link change for a project, add that issue as an example here


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.9. Update the Test Site link

## Template
````
- [  ] Under the `links` variable, find `name: Test Site` and change the following:

**From:**
```
  - name: Test Site
    url: `[Insert current URL]`
```
**To:**
```
  - name: Test Site
    url: `[Insert desired URL]`
```
````

## Example
- **[PENDING]** When there is a Test Site link change for a project, add that issue as an example here


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.10. Update Demo Site link 

## Template
````
- [  ] Under the `links` variable, find `name: Demo Site` and change the following:

**From:**
```
  - name: Demo Site
    url: `[Insert current URL]`
```
**To:**
```
  - name: Demo Site
    url: `[Insert desired URL]`
```
````

## Example
- **[PENDING]** When there is a Demo Site link change for a project, add that issue as an example here


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.11. Update Site link

## Template
````
- [  ] Under the `links` variable, find `name: Site` and change the following:

**From:**
```
  - name: Site
    url: `[Insert current URL]`
```
**To:**
```
  - name: Site
    url: `[Insert desired URL]`
```
````

## Example
- **[PENDING]** When there is a Site link change for a project, add that issue as an example here


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 3.12. Leadership
## 3.12.1. Remove person(s) and info from leadership
### Template
````
- [ ] Find the `leadership` variable and remove the following:
```
[Insert the lines of code to remove the specified leadership member]
```
````
### Example
- https://github.com/hackforla/website/issues/6632

## 3.12.2. Add person(s) and info to leadership
### Template
````
- [ ] Find the `leadership` variable and add the following profile. If there are other [Insert role]s, place [Insert leadership member's name]'s profile after the other [Insert role]s' profiles.
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
````

### Example
- https://github.com/hackforla/website/issues/6632


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
```
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/expunge-assist.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [Insert relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the Expunge Assist page [^2] in your local environment and include before and after screenshots with your pull request

### Resources/Instructions
- This issue was initiated by #5627
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project page URL: https://www.hackforla.org/projects/expunge-assist
```

**Comment by JessicaLucindaCheng** [comment]
```
Update Project Profile: Expunge Assist (Remove)
```

```
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/expunge-assist.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] Under the `leadership` section, remove
```
```
- [ ] Verify the changes by viewing the Expunge Assist page [^2] in your local environment and include before and after screenshots with your pull request

### Resources/Instructions
- This issue was initiated by #5627
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project page URL: https://www.hackforla.org/projects/expunge-assist
```

**Comment by JessicaLucindaCheng** [comment]
```
Update Project Profile: Expunge Assist (Add)
```

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/expunge-assist.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] Find the `leadership` variable and add the following as a new entry:
```
  - name: [Insert leadership member's name]
    github-handle: [Insert GitHub handle]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
- [ ] Verify the changes by viewing the Expunge Assist page [^2] in your local environment and include before and after screenshots with your pull request

### Resources/Instructions
- This issue was initiated by #5627
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project page URL: https://www.hackforla.org/projects/expunge-assist
````

**Comment by JessicaLucindaCheng** [comment]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[INSERT filename].md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [INSERT relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] [INSERT name of project] page [^2]
  - [ ] the project's card on the Projects' page [^3]   [If the ONLY changes are for the `leadership` variable, REMOVE this action item to check the project's card.]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency** label.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency</b> label</summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [INSERT project specific page URL here]
[^3]: Projects page URL: https://www.hackforla.org/projects/ [If the ONLY changes are for the `leadership` variable, REMOVE this resource to the Projects page.]

**Comment by JessicaLucindaCheng** [comment]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[INSERT filename].md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [INSERT relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] 311 Data page [^2]
  - [ ] the project's card on the Projects' page [^3]   [If the ONLY changes are for the `leadership` variable, REMOVE this action item to check the project's card.]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/311-data
[^3]: Projects page URL: https://www.hackforla.org/projects/ [If the ONLY changes are for the `leadership` variable, REMOVE this resource to the Projects page.]

**Comment by JessicaLucindaCheng** [comment]
# 3.13. Change project status
## Template
````
- [ ] Find the `status` variable and change the status

**From**
```
status: [Insert current status]
```
**To**
```
status: [Insert desired status]
```
````

## Example
- #6064 


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
````
Update Project Profile: Access the Data ()
````

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/access-the-data.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [INSERT relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] Access the Data page [^2]
  - [ ] the project's card on the Projects' page [^3]   [If the ONLY changes are for the `leadership` variable, REMOVE this action item to check the project's card.]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/access-the-data
[^3]: Projects page URL: https://www.hackforla.org/projects/ [If the ONLY changes are for the `leadership` variable, REMOVE this resource to the Projects page.]
````

**Comment by JessicaLucindaCheng** [comment]
```
Update Communities of Practice: [Insert community's name] ([Insert short description of change])
```

````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_data/internal/communities/[INSERT filename].yml` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [INSERT relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request: the [Insert community's name] section on the Communities of Practice page [^2]
- [ ] Once your pull request is merged, go to #[Insert ER number that initiated this issue] and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #[Insert ER number that initiated this issue] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #[Insert ER number]
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Communities of Practice page URL: https://www.hackforla.org/communities-of-practice
````


### Add leadership member
````
- [ ] Find the `leadership` variable and add the following as a new entry:
```
  - name: [Insert leadership member's name]
    role: [Insert role]
    links:
      slack: https://hackforla.slack.com/team/[Insert Slack member ID]
      github: https://github.com/[Insert GitHub handle]
    picture: https://avatars.githubusercontent.com/[Insert GitHub handle]
```
````


### Remove leadership member
````
- [ ] Find the `leadership` variable and remove the following:
```
[Insert the lines of code to remove the specified leadership member]
```
````

**Comment by JessicaLucindaCheng** [comment]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/food-oasis.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [INSERT relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] Food Oasis page [^2]
- [ ] Once your pull request is merged, go to #6285  and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #6285 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6285 
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/food-oasis

**Comment by JessicaLucindaCheng** [comment]
# 4. Add labels
Apply the following:
- A complexity label
  - Either a **good first issue** label or **Complexity: Small** label
- Feature label(s)
  - **P-Feature: Project Info and Page**
  - If changes affect the [Project page](https://www.hackforla.org/projects/), add the **P-Feature: Projects page**
- **role: front end** and **role: back end/devOps**
- **size: 0.25 pt**
- **time sensitive**

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 5. Submit new issue
Select the **Submit new issue** button.

<img width="750" alt="" src="https://github.com/hackforla/website/assets/31293603/02528d27-655d-4b6d-abe8-8206a3db651c">

<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 6. Add the issue to the project board


<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
# 6. Check the issue is added to the "Project Board"

Check the new issue you just created is on the "Project Board".
<img width="250" alt="Issue is on the Project Board" src="https://github.com/hackforla/website/assets/31293603/ecfa2b89-4750-417f-801e-7b05b8a90318">


If the issue is not on the "Project Board", add it to the "Project Board".
<img width="250" alt="" src="https://github.com/hackforla/website/assets/31293603/d4624b3c-c6a4-4e93-8b02-c45b87371b34">



<br><br><br>

[Back to Table of Contents](https://github.com/hackforla/website/issues/5630#issuecomment-1740241650)

**Comment by JessicaLucindaCheng** [comment]
```
Update Project Profile: Civic Tech Jobs ([Insert short description of change])
```
```
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/civic-tech-jobs.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] [Insert relevant action items. See the rest of the guide for action items to add here.]
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] Civic Tech Jobs page [^2]
- [ ] Once your pull request is merged, go to #6583 and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #6583 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6583
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/civic-tech-jobs
```

**Comment by JessicaLucindaCheng** [comment]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/civic-tech-jobs.md` file
- [ ] Observe the existing syntax of the front matter block [^1] in the file
- [ ] Find the `leadership` variable and add the following profiles. If there are other UX/UI Designers, place [Insert Armon Brown's, Carmen Mak's, & Brenda Njemanze's profile after the other UX/UI Designers' profiles
  - [ ] Add
    ```
      - name: Armon Brown
        github-handle: stanarmon
        role: UX/UI Designer
        links:
          slack: https://hackforla.slack.com/team/U06J0UCAWHE
          github: https://github.com/stanarmon
        picture: https://avatars.githubusercontent.com/stanarmon
    ```
  - [ ] Add
    ```
      - name: Carmen Mak
        github-handle: CarmenMM1
        role: UX/UI Designer
        links:
          slack: https://hackforla.slack.com/team/U06313EMD41
          github: https://github.com/CarmenMM1
        picture: https://avatars.githubusercontent.com/CarmenMM1
    ```
  - [ ] Add
    ```
      - name: Brenda Njemanze
        github-handle: brendasdesign
        role: UX/UI Designer
        links:
          slack: https://hackforla.slack.com/team/U06MWMRSCJW
          github: https://github.com/brendasdesign
        picture: https://avatars.githubusercontent.com/brendasdesign
    ```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] Civic Tech Jobs page [^2]
- [ ] Once your pull request is merged, go to #6583 and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #6583 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6583
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/civic-tech-jobs

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng I have temporarily added the label: `Ignore: Test` so that this does not show up in the dashboard. When you are done crafting it, please remove the label.

**Comment by JessicaLucindaCheng** [comment]
- Note to self: Should a template like this be added somewhere in here or separately: https://github.com/hackforla/website/issues/5337?

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng I used the templates in this issue to create a draft issue and found that it was good for changing things that are already there, but not so good for changing things that don't exist in the .md file.  See my issue draft for examples
https://github.com/hackforla/website/issues/6579#issuecomment-2148607672

**Comment by JessicaLucindaCheng** [comment]
### Update
- **Progress:** Need to finish the initial draft for changing things in project profiles.
- **Blockers:** None
- **Availability:** TBA
- **ETA:** TBA

### Tasks
- [ ] Figure out if a template like this should be added somewhere in here or separately: https://github.com/hackforla/website/issues/5337
- [ ] Figure out what items need to be modified to include adding it if it doesn't exist. See:
  -  https://github.com/hackforla/website/issues/6579#issuecomment-2148607672
  - https://github.com/hackforla/website/issues/5630#issuecomment-2148622166

**Comment by JessicaLucindaCheng** [comment]
3. Based on the requested changes, add the relevant action items from below
  3.1. [Update project title](https://github.com/hackforla/website/issues/5630#issuecomment-1774117138) - will exist
  3.2. [Update the description](https://github.com/hackforla/website/issues/5630#issuecomment-1774119160) - will exist
  3.3. Update the image - will exist
  3.4. [Update the technologies](https://github.com/hackforla/website/issues/5630#issuecomment-1774119289)
  3.5. [Update the partner(s)](https://github.com/hackforla/website/issues/5630#issuecomment-1774119349)
  3.6. [Update the tools](https://github.com/hackforla/website/issues/5630#issuecomment-1774119429)
  3.7. [Update the GitHub link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119521) - will exist
  3.8. [Update the Slack link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119587) - will exist
  3.9. [Update the Test Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119650)
  3.10. [Update Demo Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119730)
  3.11. [Update Site link](https://github.com/hackforla/website/issues/5630#issuecomment-1774119764)
  3.12. [Leadership](https://github.com/hackforla/website/issues/5630#issuecomment-1774122890) - will exist
      3.12.1. Remove person(s) and info from leadership
      3.12.2. Add person(s) and info to leadership
  3.13. [Change project status](https://github.com/hackforla/website/issues/5630#issuecomment-1797822821) - will exist
4. Add labels
5. [Submit new issue](https://github.com/hackforla/website/issues/5630#issuecomment-2003436167)
6. [Check the issue is added to the "Project Board (hackforla/website)"](https://github.com/hackforla/website/issues/5630#issuecomment-2003566185)

- add project one sheet for stakeholder to fill out in template that they use for project updates
- add how to add and edit a one sheet in here
- see existing templates and update my template as necessary:
  - https://github.com/hackforla/website/issues/7512
  - https://github.com/hackforla/website/issues/new/choose
    - Project Profile leader updates - Add
    - Project Profile leader updates - Remove

**Comment by JessicaLucindaCheng** [comment]
# Title

Update Project Profile: [Insert project's name] Update [Insert name of link to be updated]


# Body
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `links` variable and change the following:

**From:**


```
  - name: [Insert current name of the link]
    url: [Insert current URL]
```
**To:**
```
  - name: [Insert desired name of the link]
    url: [Insert desired URL]
```
- [ ] Verify the changes by viewing the following in your local environment with Docker and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
  - [ ] the project's card on the Projects' page [^3]

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project-specific page URL here]
[^3]: Projects page URL: https://www.hackforla.org/projects/

**Comment by JessicaLucindaCheng** [comment]
# Title

Update Project Profile: [Insert project's name] Update [Insert either image or image-hero depending on the requested change]


# Body
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/[Insert filename].md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Find the `links` variable and change the following:

**From:**


```
  - name: [Insert current name of the link]
    url: [Insert current URL]
```
**To:**
```
  - name: [Insert desired name of the link]
    url: [Insert desired URL]
```
- [ ] Verify the changes by viewing the following in your local environment with Docker and include before and after screenshots with your pull request:
  - [ ] [Insert name of project] page [^2]
  - [ ] the project's card on the Projects' page [^3]

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: [Insert project-specific page URL here]
[^3]: Projects page URL: https://www.hackforla.org/projects/





# TDM example
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
The project team from TDM Calculator needs its Hero Image, Project Card image, and Overview link updated, so that the project can be properly represented on the Hack for LA website

### Action Items

#### Update Overview link
- [x] Download the PDF from here: https://github.com/hackforla/product-management/blob/master/project-one-sheets/TDM-Calculator-Product-One-Sheet.pdf
- [x] Replace the current PDF, with the same name, in this directory: `/assets/pdfs/` with the file above

#### Update Project Card image
- [x] Download the image in this drop down
   <details><summary>Project Card Image</summary>

   ![Image](https://github.com/user-attachments/assets/74c78798-0aab-4d42-905c-17c53d450ef5)
   </details> 
- [x] Name the image tdm-calculator.jpg
- [x] Replace the current project image, with the same name, in this directory: `/assets/images/projects/` with the image file

#### Update Hero Image
- [x] Download the image in this drop down
   <details><summary>Hero Image</summary>

   ![Image](https://github.com/user-attachments/assets/fb5c02db-4ede-434a-b3eb-85e0885b0646)
   </details> 
- [x] Name the image TDM-calculator-hero.png
- [x] Replace the current hero image, with the same name, in this directory: `/assets/images/projects/` with the image file

#### Verify your changes
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] TDM Calculator page [^2]
  - [x] the project's card on the Projects page [^3]   

[^2]: Project detailed info page URL: https://www.hackforla.org/projects/tdm-calculator
[^3]:  Projects page URL: https://www.hackforla.org/projects/
